src/BlockTypes/FeaturedProduct.php

```diff
@@ -267,4 +283,27 @@ protected function enqueue_data( array $attributes = [] ) {
 		$this->asset_data_registry->add( 'min_height', wc_get_theme_support( 'featured_block::min_height', 500 ), true );
 		$this->asset_data_registry->add( 'default_height', wc_get_theme_support( 'featured_block::default_height', 500 ), true );
 	}
+
+	/**
+	 * Get the default attributes values from the block.json metadata file
+	 *
+	 * @return array
+	 */
+	private function get_block_metadata_defaults(): array {
```

Comment:
You might want to take a look at this Core function, and see if we can make use of it. Gutenberg does, so we should probably align with it:

https://developer.wordpress.org/reference/functions/register_block_type_from_metadata/

Reply:
Ended up removing the `get_block_metadata_defaults` since the default attributes are already read from the `block.json` on the `register_block_type` function.